velox/exec/Operator.cpp

```diff
@@ -368,6 +368,30 @@ void Operator::recordSpillStats() {
     common::updateGlobalMaxSpillLevelExceededCount(
         lockedSpillStats->spillMaxLevelExceededCount);
   }
+
+  if (lockedSpillStats->spillReadBytes != 0) {
```

Comment:
Can you put these new metrics as consts of operator class? And update the documentation as well? Thanks!